packages/babel-plugin-transform-es2015-modules-amd/test/fixtures/amd/get-module-name-option/expected.js

```diff
@@ -1 +1,3 @@
-define("my custom module name", [], function () {});
+define("my custom module name", [], function () {
+  "use strict";
```

Comment:
Now I wonder if we can just not print the "use strict' directive if the body is empty anyway?